lib/web_ui/test/engine/semantics/text_field_test.dart

```diff
@@ -63,6 +63,29 @@ void testMain() {
     semantics().semanticsEnabled = false;
   });
 
+  test('tap detection works', () async {
+    debugBrowserEngineOverride = BrowserEngine.webkit;
+    debugOperatingSystemOverride = OperatingSystem.iOs;
+
+    final SemanticsActionLogger logger = SemanticsActionLogger();
+    semantics()
+      ..debugOverrideTimestampFunction(() => _testTime)
+      ..semanticsEnabled = true;
+
+    createTextFieldSemantics(value: 'hello');
+
+    final DomElement textField = appHostNode
+        .querySelector('input[data-semantics-role="text-field"]')!;
+
+    textField.dispatchEvent(createDomPointerEvent('pointerdown'));
+    textField.dispatchEvent(createDomPointerEvent('pointerup'));
```

Comment:
Let's pass explicit values for `clientX` and `clientY` to make sure that the delta calculation logic is also working as intended.

Reply:
Done!